python/kserve/requirements.txt

```diff
@@ -1,31 +1,31 @@
-six>=1.15
-python_dateutil>=2.5.3
-setuptools>=21.0.0
-urllib3>=1.15.1
-kubernetes>=12.0.0
-tornado>=6.0.0
+six>=1.16.0
+python_dateutil>=2.8.0
+setuptools>=60.6.0
+urllib3>=1.26.8
+kubernetes>=21.7.0
+tornado>=6.1
 argparse>=1.4.0
-minio>=4.0.9,<7.0.0
-google-cloud-storage>=1.41.1
-adal>=1.2.2
-table_logger>=0.3.5
-numpy~=1.19.2
-azure-storage-blob==12.8.1
-azure-identity>=1.6.0
+minio>=4.0.9,<=7.1.3
+google-cloud-storage>=2.1.0
+adal>=1.2.7
+table_logger>=0.3.6
+numpy~=1.21.5
+azure-storage-blob==12.9.0
+azure-identity>=1.8.0
 cloudevents>=1.2.0
-avro>=1.10.1
-boto3==1.20.24
-psutil>=5.0
+avro>=1.11.0
+boto3==1.21.0
+psutil>=5.9.0
 ray[serve]==1.9.2
 grpcio>=1.34.0
-google_api_core==1.29.0
+google_api_core==2.5.0
 jmespath==0.10.0
-googleapis_common_protos==1.53.0
-cachetools==4.2.2
-google_auth==1.34.0
-cffi==1.14.6
-cryptography==3.4.7
-idna==3.2
-certifi==2021.5.30
-azure_core==1.17.0
-tritonclient==2.14.2
+googleapis_common_protos==1.54.0
+cachetools==5.0.0
+google_auth==2.6.0
```

Comment:
we can remove pinned `google_auth` as it is depended by `google-cloud-storage`.
https://github.com/googleapis/python-storage/blob/main/setup.py#L31